rules/windows/privilege_escalation_printspooler_suspicious_spl_file.toml

```diff
@@ -3,22 +3,74 @@ creation_date = "2020/08/14"
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2022/08/24"
+updated_date = "2022/09/19"
 
 [rule]
 author = ["Elastic"]
 description = """
 Detects attempts to exploit privilege escalation vulnerabilities related to the Print Spooler service including
-CVE-2020-1048 and CVE-2020-1337. .
+CVE-2020-1048 and CVE-2020-1337.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious PrintSpooler SPL File Created"
-note = """## Threat intel
+note = """## Triage and analysis
 
-Refer to CVEs, CVE-2020-1048 and CVE-2020-1337 for further information on the vulnerability and exploit. Verify that the relevant system is patched.
+### Investigating Suspicious PrintSpooler SPL File Created
```

Comment:
Is "PrintSpooler" correct here, since it's "Print Spooler" below (and also when I googled it)?